domenetjenester/behandling/src/main/java/no/nav/foreldrepenger/behandling/impl/HendelseForBehandling.java

```diff
@@ -5,5 +5,6 @@ public enum HendelseForBehandling {
     ENHET,
     AVSLUTTET,
     OPPRETTET,
-    ANNET;
+    ANNET,
+    BRUKEROPPGAVE,
```

Comment:
Har du planlagt med BRUKEROPPGAVE i fpsak. Pt oppstår de kun i fptilbake

Reply:
Du har nok rett. Den var lagt inn med tanke på at fptilbake skulle publisere slike på samme topic - ikke direkte til fpinfo-historikk.
Så kan man sikkert legge inn en hendelse MIGRERING i fp-kontrakter for å støtte evt framtidig massemigrering.